custom/panel_templates/Default/template.php

```diff
@@ -304,6 +304,23 @@ public function onPageLoad() {
                                 AssetTree::DATATABLES,
                             ]);
 
+                            $url_parameters = [];
+                            if (isset($_GET['group'])) {
+                                $url_parameters[] = 'group=' . $_GET['group'];
```

Comment:
Should these $_GET params all be sanitised in case somebody sends a malicious link to a staff member